packages/expect/src/spyMatchers.ts

```diff
@@ -68,6 +78,40 @@ const getRightAlignedPrinter = (label: string): PrintLabel => {
     suffix;
 };
 
+type IndexedCall = [number, Array<unknown>];
+
+// Return either empty string or one line per indexed result,
+// so additional empty line can separate from `Number of returns` which follows.
+const printReceivedCalls = (
+  label: string,
+  indexedCalls: Array<IndexedCall>,
+  isOnlyCall: boolean,
+  iExpectedCall?: number,
+) => {
+  if (indexedCalls.length === 0) {
+    return '';
+  }
+
+  if (isOnlyCall) {
+    return label + printReceivedArgs(indexedCalls[0]) + '\n';
+  }
+
+  const printAligned = getRightAlignedPrinter(label);
+
+  return (
+    label.replace(':', '').trim() +
```

Comment:
Wouldn't it be cleaner to pass just 'Received' as the label and concatenate in the 1 case instead of replacing in the >1 case?

Reply:
Good eyes! It started that way, but changed to align values for negative assertions:

```test
`Expected: not ${printExpectedArgs(expected)}\n`
'Received:     '
```

when there is only one call, which must be the expected call for Last or Nth matcher:

```js
  if (isOnlyCall) {
    return label + printReceivedArgs(indexedCalls[0]) + '\n';
  }
```

Reply:
Not sure we're talking about the same thing - I meant calling it as
```js
printReceivedCalls('Received', ...)
```
and changing the two label usages to
```js
label + ':     ' + printReceivedArgs(indexedCalls[0])
```
(for `isOnlyCall`) and just
```js
label
```
(otherwise) to avoid `replace`+`trim` magic

Reply:
> Not sure we're talking about the same thing

For sure, I confused myself because the reason to split the pull request is report for negative and positive assertion is too different to share the same function!

So I will remove the argument and rename function as `printReceivedCallsNegative`

Thanks for your patience.